libmamba/include/mamba/api/update.hpp

```diff
@@ -15,7 +15,8 @@ namespace mamba
         Configuration& config,
         bool update_all = false,
         bool prune_deps = false,
-        bool remove_not_specified = false
+        bool env_update = false,           // Specific to `env update` command
```

Comment:
Potential future improvement: replace these arguments by a struct containing the sequence of booleans, that would help with maintaining the code calling this and make everything less verbose.